src/components/ApplicationPage/ApplicationPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from "react";
+import { RouteComponentProps } from "@reach/router";
+import { Application } from "../Application";
+import { GetFundingApplicationQuery } from "../../API";
+
+import { getFundingApplication } from "../../graphql/queries";
+import { useQuery } from "react-apollo-hooks";
+import gql from "graphql-tag";
+
+interface Props extends RouteComponentProps {}
+
+const GET_APPLICATION = gql(getFundingApplication);
+
+const v: GetFundingApplicationQuery = {
```

Comment:
There's no reason to use mock data in the live page. Mock it in the storybook to check designs, and use live data in the real site.